.env

```diff
@@ -0,0 +1 @@
+ENCRYPTFI_API_DATABASE_URL="postgres://encryptfi_api:admin@localhost"
```

Comment:
I believe that .env does not be commit, this show informations of our infra.

Reply:
nice, removed